resources/event-bus/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+dependencies:
+  - name: publish
+    condition: global.eventBus
```

Comment:
To make it more clear, should we consider calling it `global.not-knative`

Reply:
I would go for more like`kyma`/`built-in` but not the `not-`, what do you think?

Reply:
`global.kyma-default` as knative is enabled with a feature flag.